### ISSUE TITLE
undefined ctx.me when webhook used

### ISSUE THREAD

**Issue by udamir** [telegraf/telegraf#1184]
I have created telegram bot based on **telegraf 3.21** with webhook and when deployed it on heroku and I found out that ```ctx.me``` is empty and bot doesn't now his name). When I test it on localhost with polling update everything works fine. 

```ts
const app = express()

if (MODE === "webhook") {
  // Start webhook via launch method (preffered)
  bot.telegram.setWebhook(`${WEBHOOK_URL}/bot${TOKEN}`)
  app.use(bot.webhookCallback("/bot" + TOKEN))
  logger.info(`Webhook on ${WEBHOOK_URL}/bot${TOKEN}`)
} else {
  bot.launch().then(() => {
    logger.info("Bot started")
  }).catch((error) => {
    logger.error("Cannot start bot", error)
  })
}
```
Any ideas what I can do wrong with webhook?

**Comment by wojpawlik** [comment]
As of 3.38, you can use `.launch` for webhook as well, and it will set `ctx.me`.

`.webhookCallback` doesn't set `ctx.me` because it's synchronous, see #1072.

**Comment by udamir** [comment]
Thank you!

**Comment by KnorpelSenf** [comment]
Will be fixed by #1200
